ecs-agent/api/ecs/client/ecs_client.go

```diff
@@ -506,6 +506,11 @@ func (client *ecsClient) SubmitTaskStateChange(change ecs.TaskStateChange) error
 }
 
 func (client *ecsClient) submitTaskStateChange(change ecs.TaskStateChange) error {
+
```

Comment:
nit: Unnecessary newline added here - not blocking to this pull request, but please make sure this is removed in a followup.

Reply:
Sure.